### PR TITLE
UI changes in patient lookup

### PR DESCRIPTION
**Pull request by saket**

### CODE REVIEW

File: app/src/main/res/layout/screen_patient_search.xml [hunk 1]
```diff
@@ -1,65 +1,150 @@
...
-    tools:listitem="@layout/list_patient_search" />
+    android:layout_toEndOf="@+id/patientsearch_back"
+    android:scrollbarStyle="insideOverlay"
+    android:scrollbars="vertical">
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="vertical"
+      android:paddingBottom="64dp"
```
Comment (vinaysshenoy) [inline app/src/main/res/layout/screen_patient_search.xml]: Should this be extracted to a dimension resource?

File: app/src/main/res/layout/screen_patient_search.xml [hunk 2]
```diff
@@ -1,65 +1,150 @@
...
+      android:paddingEnd="@dimen/spacing_8"
+      android:paddingTop="@dimen/spacing_16"
+      tools:ignore="RtlSymmetry">
+
+      <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <android.support.design.widget.TextInputLayout
```
Comment (vinaysshenoy) [inline app/src/main/res/layout/screen_patient_search.xml]: `TextInputLayout` is a sub-class of `LinearLayout`. Can't we add the clear button as a child of the `TextInputLayout` itself?

File: app/src/main/res/layout/screen_patient_search.xml (hunk 1 above)
Comment (saket) [reply app/src/main/res/layout/screen_patient_search.xml]: Could, sure

File: app/src/main/res/layout/screen_patient_search.xml (hunk 2 above)
Comment (saket) [reply app/src/main/res/layout/screen_patient_search.xml]: No, because the clear button needs to appear in front of the text field

**Pull request merged**